backend/coins/btc/maketx/maketx.go

```diff
@@ -168,7 +168,11 @@ func NewTxSpendAll(
 		TxOut:    []*wire.TxOut{output},
 		LockTime: 0,
 	}
-	txsort.InPlaceSort(unsignedTransaction)
+
+	// Shuffle inputs and outputs
+	ShuffleTxInputs(unsignedTransaction)
```

Comment:
small nit: looks like you don't really need two separate functions, right?

Reply:
I agree, having just one function is sufficient, I've made the update, PTAL